upstream/composer.json

```
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": ">=7.3",
```

```suggestion
        "php": ">=7.4",
```
We should also be matching Drupal 9.4.x's minimum PHP version